02-using-Shiny.Rmd

```diff
@@ -41,33 +43,37 @@ This checks the format of your workflow files to make sure you have a valid file
+
 
 ### Abort a workflow
-Sometimes you realize you might just want to kill a workflow.  Using the workflow submission id, you can specifically kill workflows by using this box.  Note it will take Cromwell some time to coordinate SLURM job cancellations but it will clean everything up for you, it's just not instant. 
+Sometimes you realize you want to kill a workflow.  Using the workflow submission id, you can kill specific workflows using this box.  Note it will take Cromwell some time to coordinate SLURM job cancellations, but it will clean everything up for you. 
 <img src="assets/cromwell/abort.png" title="Abort a workflow" alt="Crowmell app" style="display: block; margin: auto;" />
 
 ### Troubleshoot a workflow
-Especially in the beginning if you have catastrophic workflow failures and you can't even figure out what's going on, you can come back to this Troubleshoot box to retrieve the entire, unformatted JSON output of all metadata Cromwell has about your workflow here.  You probably are better served by the next tab in the app for checking up on how your workflow is going, but if there's nothing there that's helpful, then this box is where you'll want to go.  
-> Note: this is not for the faint of heart, but it will give you hints once you get used to understanding what Cromwell is telling you.  
+Especially in the beginning if you have catastrophic workflow failures and you can't even figure out what's going on, you can come back to this Troubleshoot box to retrieve the entire, unformatted JSON output of all metadata Cromwell has about your workflow.  You probably are better served by the "Track Jobs" tab for checking how your workflow is going, but if there's nothing there that's helpful, then this box is where you'll want to go.  
+> Note: this output is not for the faint of heart, but it will give you hints once you get used to understanding what Cromwell is telling you.  
 <img src="assets/cromwell/troubleshoot.png" title="Troubleshoot a workflow" alt="Crowmell app" style="display: block; margin: auto;" />
 
 
 ## Track Jobs Tab
 
 Once you've submitted a workflow, you'll want to track how it's going in the Track Jobs tab. 
 
-### Workflows of History
-Here you'll see that you can query for as many days of history worth of workflows that you'd like, filter that result (for instance if you have submitted a LOT of workflows and the app is slow) for workflows with a specific name or with specific status(es) (such as failed, succeeded, etc). 
+### History of workflows
+At the top, you'll see that you can display as many days of workflow history as you'd like, filter that result for workflows with a specific name or with specific status(es) like 'failed', 'succeeded', etc.  This can help if you have submitted a LOT of workflows and the app is slow.
```

Comment:
Actually it's not the app that's slow, it's your Cromwell server working to keep track of all the bits and bobbles of workflow status'.  So maybe change the last sentence to: "This can help if you have submitted a LOT of workflows and you don't want to see them all, or if the Cromwell server is still busy working through all of your submissions and recording the status'"